app/services/activitypub/fetch_remote_status_service.rb

```diff
@@ -44,4 +44,9 @@ def supported_context?
   def expected_type?
     %w(Note Article).include? @json['type']
   end
+
+  def needs_update(actor)
+    # TODO: optimize by ruling out accounts that have local followers
```

Comment:
I think the full account object may actually be present in the fetched json, but I'm not too sure it's mandated by the spec. If it is, it could be considered prefetched, avoiding further queries, and we could completely bypass the daily limit this way.

Reply:
* It is not mandatory.
* You should _always_ fetch the account (or _dereference_ the ID in the language of the spec) to make sure the ID actually points to the account.

Reply:
In that case, the account is already known, and the json is already considered authoritative for the author's toot, so I'm tempted to just use that info instead of making a new request…
But I will just trigger an account refresh for now, it can be optimized later on.